interface/src/lib.rs

```diff
@@ -1,5 +1,75 @@
-//! The [system native program][np].
+//! The [system native program interface][np].
```

Comment:
We should check that rustdoc is happy in CI (currently it shows some warnings):
```
cd interface
RUSTDOCFLAGS="--cfg docsrs -D warnings" cargo +nightly doc --all-features --no-deps
````